indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -317,8 +317,8 @@
 
           ;; Temporal range of min and max granule values or the same as collection start and end date
           ;; if the collection has not granules.
-          :granule-start-date             m/date-field-mapping
-          :granule-end-date               m/date-field-mapping
+          :granule-start-date             (m/stored m/date-field-mapping)
```

Comment:
We can't change Elasticsearch fields once they've been created. We have to add new fields, keep these existing fields, and change our indexing and search code to make sure they are using the new fields.

i.e.:
```
          :granule-start-date             m/date-field-mapping
          :granule-end-date               m/date-field-mapping
          :granule-start-date-stored (m/stored m/date-field-mapping)
          :granule-end-date-stored   (m/stored m/date-field-mapping)
```

Reply:
Done.